lib/Remotery.c

```diff
@@ -6115,6 +6105,11 @@ static rmtError Remotery_SendSampleTreeMessage(Remotery* rmt, Message* message)
     error = bin_SampleTree(bin_buf, sample_tree);
     rmt_EndCPUSample();
 
+    if (g_Settings.sampletree_handler != NULL)
+    {
+        g_Settings.sampletree_handler(rmt, g_Settings.sampletree_context, sample_tree);
+    }
```

Comment:
I had to put it before the `FreeSamples()` as the sample pointers would be invalid after that call.